### PR TITLE
PHOENIX-5689: TableNotFoundException in IndexUpgradeTool in case of v…

### PR DESCRIPTION
**Pull request by swaroopak**
…iew indexes

### CODE REVIEW

**Comment by swaroopak** [comment]
@priyankporwal @gokceni @tkhurana

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexUpgradeToolIT.java [hunk 1]
```diff
@@ -19,76 +19,121 @@
...
             int countViews = 0;
             List<String> views = new ArrayList<>();
-            List<Integer> indexCount = new ArrayList<>();
+            List<String> tenants = new ArrayList<>();
             while (rs.next()) {
                 views.add(rs.getString(1));
+                if(multiTenant) {
+                    Assert.assertNotNull(rs.getString(2));
+                }
+                tenants.add(rs.getString(2));
```
Comment (gokceni) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexUpgradeToolIT.java]: Will this be adding null for multiTenant=false?

File: phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java [hunk 2]
```diff
@@ -652,51 +652,15 @@ private int startIndexRebuilds(Connection conn,
                     (GLOBAL_INDEX_ID.equals(tenantId)?"":"_"+tenantId) +"_"
                     + UUID.randomUUID().toString();
             String[] args = getIndexToolArgValues(schema, baseTable, indexName, outFile, tenantId);
-            Connection newConnection = conn;
-            Connection tenantConnection = null;
             try {
                 LOGGER.info("Rebuilding index: " + StringUtils.join( args,","));
                 if (!dryRun) {
-                    // If the index is in DISABLED state, indexTool will fail.
-                    // First to ALTER REBUILD ASYNC.
```
Comment (gjacoby126) [inline phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java]: this is now taken care of within the index tool?

Comment (gokceni) [reply phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java]: I am doing PHOENIX-5512 right now to take care of this

**Pull request merged**